Documentation.md

```diff
@@ -0,0 +1,265 @@
+## Arctica Greeter - Installation Guide for Ubuntu <xml> 16.04 LTS (Latest version)<xml/> ##
+
+#<span style="color:darkblue; font-family: Cambria;"> Introduction <span/>#

```

Comment:
Please drop all formatting elements from this .md file. The file should be easily editable and readable as plaintext. Thanks!